### PR TITLE
Update changelog for release (v0.17.0)

### PR DESCRIPTION
**Pull request by mccurdyc**
:star2:  [Rendered CHANGELOG](https://github.com/terraform-providers/terraform-provider-fastly/pull/266/files?short_path=4ac32a7#diff-4ac32a78649ca5bdd8e0ba38b7006a1e)

cc @phamann

### CODE REVIEW

**Comment by mccurdyc** [comment]
Force-pushed a format / order fix.

**Comment by mccurdyc** [comment]
Force-pushed another order fix.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+* resource/fastly_service_v1/logging_loggly: Add Loggly logging support ([#241](https://github.com/terraform-providers/terraform-provider-fastly/pull/241))
+* resource/fastly_service_v1/logging_sftp: Add SFTP logging support ([#236](https://github.com/terraform-providers/terraform-provider-fastly/pull/236))
+* resource/fastly_service_v1/logging_ftp: Add FTP logging support ([#235](https://github.com/terraform-providers/terraform-provider-fastly/pull/235))
+* resource/fastly_service_v1/logging_elasticsearch: Add Elasticsearch logging support ([#234](https://github.com/terraform-providers/terraform-provider-fastly/pull/234))
+
+NOTES:
+
+* resource/fastly_service_v1/httpslogging: Use `trimSpaceStateFunc` to trim leading and trailing whitespace from the `tls_ca_cert`, `tls_client_cert` and `tls_client_key` fields ([#264](https://github.com/terraform-providers/terraform-provider-fastly/pull/264))
+* resource/fastly_service_v1/splunk: Use `trimSpaceStateFunc` to trim leading and trailing whitespace from the `tls_ca_cert` field ([#264](https://github.com/terraform-providers/terraform-provider-fastly/pull/264))
```
Comment (mccurdyc) [inline CHANGELOG.md:23]: @phamann These could introduce a new diff of state for customers. Would remove a new line.

I don't think this is technically a breaking change, but does introduce a state change just from updating your provider version.

Just something to think about.

**Comment by mccurdyc** [comment]
I'll add a line in the NOTES with links to PRs updating docs.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,30 @@
-## 0.17.0 (Unreleased)
+## 0.18.0 (Unreleased)
+
+## 0.17.0 (June 19, 2020)
```
Comment (phamann) [inline CHANGELOG.md]:
```suggestion
## 0.17.0 (June 22, 2020)
```

**Review by phamann — CHANGES_REQUESTED** [review]
LGTM other than the date

**Review by phamann — APPROVED** [review]
:+1: LGTM

**Pull request merged**